### PR TITLE
修改添加ApolloPropertySources到env中的逻辑，添加列表判断操作

### PR DESCRIPTION
**Pull request by kangjiabang**
具体见issue:https://github.com/ctripcorp/apollo/issues/2731

### CODE REVIEW

**Comment by kangjiabang** [comment]
@JaredTan95 ,please review

File: apollo-client/src/main/java/com/ctrip/framework/apollo/spring/config/PropertySourcesProcessor.java [hunk 1]
```diff
@@ -82,17 +82,20 @@ private void initializePropertySources() {
...
 
-    // add after the bootstrap property source or to the first
-    if (environment.getPropertySources()
-        .contains(PropertySourcesConstants.APOLLO_BOOTSTRAP_PROPERTY_SOURCE_NAME)) {
+    // ensure ApolloBootstrapPropertySources is still the first
+    ensureBootstrapPropertyPrecedence(environment);
 
-      // ensure ApolloBootstrapPropertySources is still the first
-      ensureBootstrapPropertyPrecedence(environment);
+    if (composite.getPropertySources() != null && composite.getPropertySources().size() > 0) {
```
Comment (nobodyiam) [inline apollo-client/src/main/java/com/ctrip/framework/apollo/spring/config/PropertySourcesProcessor.java]: how about using `return early pattern` which keeps the code a little bit clean?

```java
  if (CollectionUtils.isEmpty(composite.getPropertySources())) {
      return;
  }
```

Comment (kangjiabang) [reply apollo-client/src/main/java/com/ctrip/framework/apollo/spring/config/PropertySourcesProcessor.java]: it seems a good advice.

Comment (nobodyiam) [reply apollo-client/src/main/java/com/ctrip/framework/apollo/spring/config/PropertySourcesProcessor.java]: It looks good to me. However, the code style is not the same, e.g. Apollo's code style uses 2 spaces as indent, would you please use [apollo-code-style](https://github.com/ctripcorp/apollo/tree/master/apollo-buildtools/style) and format the changes?

Comment (kangjiabang) [reply apollo-client/src/main/java/com/ctrip/framework/apollo/spring/config/PropertySourcesProcessor.java]: i have format the code using the [apollo-code-style](https://github.com/ctripcorp/apollo/tree/master/apollo-buildtools/style) you provide

**Pull request merged**